packages/filebrowser/src/opendialog.ts

```diff
@@ -246,6 +259,11 @@ namespace Private {
       refreshInterval: options.refreshInterval,
       filterDirectories
     });
+
+    if (defaultPath) {
+      model.cd(defaultPath);
```

Comment:
Ideally this would get an `await` to ensure that the path change has enough time to propagate in the model (especially if implementation of `cd` changes in the future). For that to happen the `createFilteredFileBrowser` function will need to be changed to `async` and respective changes propagated up to where it is used, for example instead of:

```
layout.addWidget(this._browser);
```

we will want to call widget on value the promise resolves to. Similarly in `getOpenFiles` (which fortunately is already async) we should only initialise `Dialog` once `OpenDialog` is ready. We could add a public `OpenDialog.ready: Promise<void>` to know if it is ready.